ios/objc/apis/TorchModule.h

```diff
@@ -0,0 +1,34 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@class TorchIValue;
+@interface TorchModule : NSObject
+
+/**
+ Load the torchscript model and returns a TorchModule object.
+ see https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html
+ @param modelPath File path to the torchscript model
+ @return TorchModuel object
+ */
++ (TorchModule* _Nullable)loadTorchscriptModel:(NSString* _Nullable)modelPath;
```

Comment:
naming: does it need to be a long name like this or is it always prefixed with the class name? if so - why not just `[TorchModule load]`?

Reply:
Unfortunately, +load is a life-cycle method in Objective-C, we don't usually override this method, unless doing some crazy hacking stuff.